code/_onclick/hud/alerts.dm

```diff
@@ -3,7 +3,7 @@
 //PUBLIC -  call these wherever you want
 
 
-/mob/proc/throw_alert(category, type, severity, obj/new_master, override = FALSE)
+/mob/proc/throw_alert(category, type, severity, obj/new_master, override = FALSE, mob/initiator)
```

Comment:
I'm not exactly happy you did it like this, either. Why not just port how tg does it, with some adjustments instead of adding a random param to a proc that probably shouldn't have this as a param?

Reply:
Why shouldn't it have it? It's just a reference to the mob that began the interaction.


I don't read through /tg/ to see what they have. I was using a system that already existed as I don't like complicating things for the next person that may come along.

Reply:
This proc is not used for *just* interactions, that is why